server/src/main/java/io/deephaven/server/runner/DeephavenApiServer.java

```diff
@@ -167,6 +171,13 @@ public DeephavenApiServer run() throws IOException, ClassNotFoundException, Time
         return this;
     }
 
+    private void checkGlobals(ScriptSession scriptSession) {
+        scriptSession.observeScopeChanges();
+        scheduler.runAfterDelay(100, () -> {
```

Comment:
should we parameterize this, so that in case 100ms breaks things, we can turn it down/off by configuration?